app/src/main/java/dev/andrylat/app/App.java

```diff
@@ -6,21 +6,62 @@
 import java.util.stream.IntStream;
 
 public class App {
-    private static StringBuilder errors = new StringBuilder();
+    private StringBuilder errors;
+    private Scanner scanner;
+    private String brand = "";
+
+    public App() {
+        System.out.println("Hello. Enter card number for validation:");
```

Comment:
Tomorrow I will ask you to create web-service, which will do the same but as input and output you will have HTTP. What will you do: copy your code to new implementation? Or write the additional code here?

Reply:
I see no updates here.